catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/HadoopCatalogOperations.java

```diff
@@ -72,10 +71,11 @@
 
 public class HadoopCatalogOperations implements CatalogOperations, SupportsSchemas, FilesetCatalog {
 
+  public static final String DEFAULT_FS = "fs.defaultFS";
```

Comment:
It's weird you have to same name `DEFAULT_FS` defined in your PR, it is hard to review, please rename one to make it easy to understand.

Reply:
OK

Reply:
I have changed  `DEFAULT_FS` in `HadoopCatalogPropertiesMetadata` to `DEFAULT_FS_PROVIDER`.